src/main/java/com/minecolonies/coremod/entity/pathfinding/AbstractPathJob.java

```diff
@@ -718,6 +719,9 @@ protected final boolean walk(@NotNull final Node parent, @NotNull final BlockPos
                 //  Early out on previously visited and closed nodes
                 return false;
             }
+
+            // Update change in position
+            dPos = dPos.add(0, yDelta, 0);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Introduce a new variable instead of reusing the parameter "dPos". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1226)